### PR TITLE
copr: vectorize InetNtoa

### PR DESCRIPTION
**Pull request by MaiCw4J**
Signed-off-by: mak <610546069@qq.com>

PCP #5751 
###  What have you changed?

Add vectorized InetNtoa

###  What is the type of the changes?

Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?
Please select the tests that you ran to verify your changes:
- Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No. It should not change the behavior.

###  Does this PR affect `tidb-ansible`?

No

### CODE REVIEW

**Comment by MaiCw4J** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/79

File: components/tidb_query/src/rpn_expr/impl_miscellaneous.rs [hunk 1]
```diff
@@ -25,6 +26,21 @@ pub fn is_ipv4_compat(addr: &Option<Bytes>) -> Result<Option<i64>> {
...
+#[rpn_fn]
+#[inline]
+pub fn inet_ntoa(input: &Option<Int>) -> Result<Option<Bytes>> {
+    Ok(input.as_ref().and_then(|input| {
+        if *input < 0 || *input > i64::from(u32::max_value()) {
+            None
+        } else {
+            let v = *input as u32;
+            let ipv4_addr =
+                Ipv4Addr::new((v >> 24) as u8, (v >> 16) as u8, (v >> 8) as u8, v as u8);
```
Comment (rleungx) [inline components/tidb_query/src/rpn_expr/impl_miscellaneous.rs]: How about using `Ipv4Addr::from`?

Comment (MaiCw4J) [reply components/tidb_query/src/rpn_expr/impl_miscellaneous.rs]: That's great, thank for help

**Comment by breeswish** [comment]
I guess this PR is duplicated to https://github.com/tikv/tikv/pull/5900

**Comment by MaiCw4J** [comment]
duplicated #5900

**Pull request closed without merging**